source/extensions/transport_sockets/tls/context_config_impl.cc

```diff
@@ -282,7 +308,24 @@ unsigned ContextConfigImpl::tlsVersionFromProto(
 const unsigned ClientContextConfigImpl::DEFAULT_MIN_VERSION = TLS1_VERSION;
 const unsigned ClientContextConfigImpl::DEFAULT_MAX_VERSION = TLS1_2_VERSION;
 
-const std::string ClientContextConfigImpl::DEFAULT_CIPHER_SUITES =
+// FIPS configuration
+const std::string ClientContextConfigImpl::DEFAULT_FIPS_CIPHER_SUITES =
```

Comment:
Both server and client default cipher suites look the same. I think they are going to stay this way too. Any reason not to move these to the base class then?

Reply:
With the exception of the TLS max they are but this just follows the upstream behaviour which I'm not changing